Formula/go-jira.rb

```diff
@@ -0,0 +1,31 @@
+class GoJira < Formula
+  desc "Simple jira command-line client in Go"
+  homepage "https://github.com/Netflix-Skunkworks/go-jira"
+  url "https://github.com/Netflix-Skunkworks/go-jira/archive/v1.0.12.tar.gz"
+  sha256 "7ce5d6dcc5021f2019b0107c4ad342979d18bb110188b810017b2d94ff384a33"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    dir = buildpath/"src/gopkg.in/Netflix-Skunkworks/go-jira.v1"
+    dir.install buildpath.children
+
+    ldflags = ["-w", "-s"]
+    cd dir do
+      system "go", "build", "-o", bin/"jira", "-ldflags", ldflags.join(" "), "cmd/jira/main.go"
+    end
+  end
+
+  test do
+    system "#{bin}/jira", "export-templates"
+    template_dir = Dir.home + "/.jira.d/templates/"
+    Dir.mkdir(template_dir, 700) unless Dir.exist?(template_dir)
```

Comment:
This is a clean directory, so, if you're creating the files they'll exist. If you aren't, they won't. Don't need logic to determine what you already know.

Reply:
it made a difference in my local testing (and re-testing) so I kept it. but I'll remove it here

Reply:
What was happening for you locally?

Reply:
when I was repeatedly running the tests (and trying to programatically build out template_dir) I was getting either "directory already exists" or "directory doesn't exist" errors. It's quite possible I fixed the underlying cause (user error) and the statement is no longer necessary